### PR TITLE
fix: tabs sticky and mobile bug

### LINKED ISSUE #505: Page tabs should be sticky when scrolling
### To do

-[ ] Verify scroll padding (the CSS variable) used for anchor link scroll-tos

(We also have an example of a JS scroll event for the catalog sticky filter if we need do to anything in JS too, not just CSS.)

### Browser

_No response_

### Description

image

The page tabs (Usage, Design, etc.) should be sticky, so when scrolled, they are always visible like:

image

### Steps to reproduce

See above.

### PR DESCRIPTION
**Pull request by andreancardona**
Closes #505   &  #643 

- This PR makes sure tabs are sticky 
- It also makes sure the sudo element expands full vw at all. breakpoints

NOTE: thank you @francinelucca for helping me on this one!!  💯 

## TESTING:

- [ ] make sure sticky tabs stay sticky
- [ ] make sure there are no visual regressions with tabs at any breakpoint

### CODE REVIEW

File: services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].module.scss [hunk 1]
```diff
@@ -43,3 +44,28 @@
     width: 50%;
   }
 }
+
+.tabs-column {
+  position: sticky;
+  z-index: 9000;
```
Comment (francinelucca) [inline services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].module.scss:50]: you can import like: `@use '@carbon/react/scss/utilities/z-index' as z-index;`
```suggestion
  z-index: z-index.z('overlay');;
```

**Review by francinelucca — CHANGES_REQUESTED** [review]
minor CRs.

it doesn't look like our calculations are working for the line span on max breakpoint 😢 , let's sync on monday 
image

**Review by francinelucca — APPROVED** [review]
💃

**Review by alisonjoseph — APPROVED** [review]
🔥

**Pull request merged**